src/main/java/uk/ac/ebi/embl/flatfile/writer/embl/EmblReducedFlatFileWriter.java

```diff
@@ -0,0 +1,104 @@
+package uk.ac.ebi.embl.flatfile.writer.embl;
+
+import java.io.IOException;
+import java.io.Writer;
+import java.util.HashSet;
+
+import uk.ac.ebi.embl.api.entry.Entry;
+import uk.ac.ebi.embl.api.entry.feature.Feature;
+import uk.ac.ebi.embl.api.entry.feature.SourceFeature;
+import uk.ac.ebi.embl.api.entry.reference.Reference;
+import uk.ac.ebi.embl.flatfile.EmblTag;
+import uk.ac.ebi.embl.flatfile.writer.EntryWriter;
+import uk.ac.ebi.embl.flatfile.writer.WrapType;
+import uk.ac.ebi.ena.taxonomy.taxon.Taxon;
+
+/** Reduced flat file writer.
+ */
+public class EmblReducedFlatFileWriter extends EntryWriter {
+
+    public EmblReducedFlatFileWriter(Entry entry) {
+        super(entry);
+        wrapType = WrapType.EMBL_WRAP;
+    }
+
+    private boolean excludeSource = true;
+
+    public boolean isExcludeSource() {
+        return excludeSource;
+    }
+
+    public void setExcludeSource(boolean excludeSource) {
+        this.excludeSource = excludeSource;
+    }
+
+    public final static String SEPARATOR_LINE = EmblTag.XX_TAG  + "\n";
+    public final static String TERMINATOR_LINE = EmblTag.TERMINATOR_TAG  + "\n";
+
+    public boolean write(Writer writer) throws IOException {
+        if (entry == null) {
+            return false;
+        }
+        if(new IDWriter(entry).write(writer)) {
+            writer.write(SEPARATOR_LINE);
+        }
+
+        //Not expanded entry
+        if( null == entry.getSequence()
+                || null == entry.getSequence().getSequenceByte()||entry.isNonExpandedCON() )
+            writeFeatures( writer );
+
+        if(new COWriter(entry, wrapType).write(writer)) {
+            if (entry.getSequence() != null &&
+                    entry.getSequence().getSequenceByte() != null&&!entry.isNonExpandedCON()) {
+                writer.write(SEPARATOR_LINE);
+            }
+        }
+
+        //Expanded entry
+        if( null != entry.getSequence()
+                && null != entry.getSequence().getSequenceByte()&&!entry.isNonExpandedCON() )
+            writeFeatures( writer );
```

Comment:
- We should always attempt to write features. The if statement is unnecessary.
- writeFeatures() is called twice but there should only ever be one call to it. This is the second call.